functions/android.js

```diff
@@ -9,41 +9,40 @@ module.exports = {
     };
     let updateRateLimits = true;
 
-    if (req.body.data){
-
+    if (req.body.data) {
       // Handle the web actions by changing them into a format the app can handle
       // https://www.home-assistant.io/integrations/html5/#actions
       if (req.body.data.actions) {
-        for (let i = 0; i < req.body.data.actions.length; i++) {
-          const action = req.body.data.actions[i];
-          if (action.action){
-            payload.data["action_"+(i+1)+"_key"] = action.action;
+        req.body.data.actions.forEach((action, i) => {
+          const actionIndex = i + 1;
+          if (action.action) {
+            payload.data[`action_${actionIndex}_key`] = action.action;
           }
           if (action.title) {
-            payload.data["action_"+(i+1)+"_title"] = action.title;
+            payload.data[`action_${actionIndex}_title`] = action.title;
           }
-          if (action.uri){
-            payload.data["action_"+(i+1)+"_uri"] = action.uri;
+          if (action.uri) {
+            payload.data[`action_${actionIndex}_uri`] = action.uri;
           }
-          if (action.behavior){
-            payload.data["action_"+(i+1)+"_behavior"] = action.behavior;
+          if (action.behavior) {
+            payload.data[`action_${actionIndex}_behavior`] = action.behavior;
           }
-        }
+        });
       }
 
       // Allow setting of ttl
       // https://firebase.google.com/docs/reference/admin/node/admin.messaging.AndroidConfig.html#optional-ttl
-      if (req.body.data.ttl){
+      if (req.body.data.ttl) {
         payload.android.ttl = req.body.data.ttl;
       }
 
       // https://firebase.google.com/docs/reference/admin/node/admin.messaging.AndroidConfig.html#optional-priority
-      if (req.body.data.priority){
+      if (req.body.data.priority) {
         payload.android.priority = req.body.data.priority;
       }
 
       // https://firebase.google.com/docs/reference/admin/node/admin.messaging.AndroidNotification.html
-      for (const key of [
+      const androidNotificationKeys = [
```

Comment:
For the future, this should be defined statically in the root of the module; there is no need to redefine this.